runtime/util/finish_rewriting.cpp

```diff
@@ -25,7 +24,7 @@ int32_t get_exit_code(block *);
 void init_outputs(char const *output_filename) {
   output_file = fopen(output_filename, "a");
   if (proof_output) {
-    proof_writer = new kllvm::proof_trace_file_writer(output_file);
+    proof_writer = new proof_trace_file_writer(output_file);
```

Comment:
Is there a reason for not using `kllvm` namespace anymore?

Reply:
The `proof_trace_file_writer` class is now part of the runtime library that does not use any namespaces so I followed that convention. The reason it is part of the runtime library and not the binary library as it was before is that its methods need to call other functions that are part of the runtime library and therefore there would be a cyclical dependency if it remained in the binary library.

Reply:
Make sense! Thanks!